src/MiNET/MiNET/Utils/IO/HighPrecisionTimer.cs

```diff
@@ -68,7 +68,7 @@ public class HighPrecisionTimer : IDisposable
 		public long Avarage = 0;
 
 
-		public HighPrecisionTimer(int interval, Action<object> action, bool useSignaling = false, bool skipTicks = true)
+		public HighPrecisionTimer(int interval, Action<object> action, bool useSignaling = false, bool skipTicks = true, bool highPrecision = false)
```

Comment:
Highprecision should default to true

Reply:
Yes, I think you're right

but then the question is whether we need high precision in SkyLightCalculations.Calculate
![image](https://github.com/NiclasOlofsson/MiNET/assets/16388979/4db13f15-d705-4893-8ed5-4fda746ee325)

and may I set the highPrecision parameter from Config in the same way as it is done in Level for RakConnection SendTick timer? or does it need a different configuration name?